.ci/setup.sh

```diff
@@ -34,6 +34,8 @@ setup_distro_env() {
 
 	if [[ "$ID" =~ ^opensuse.*$ ]]; then
 		script="${cidir}/setup_env_opensuse.sh"
+	elif [[ "$ID" == "alinux" ]]; then
```

Comment:
This `elif` is not necessary. `ID` being `alinux`, on line 40 it will call `setup_env_${ID}.sh` properly.